internal/webhook/inferenceservice/mutating.go

```diff
@@ -246,7 +246,7 @@ func hasOCIImagePullSecrets(obj *unstructured.Unstructured) bool {
 	if err != nil || !found {
 		return false
 	}
-	return len(imagePullSecrets) > 0 // if it is empty, we no need to delete the whoel shabang
+	return len(imagePullSecrets) > 0 // if it is empty, we don't need to delete the whole shabang
```

Comment:
Since we are already fixing typos, I will add one more.
I think it probably meant

```suggestion
	return len(imagePullSecrets) > 0 // if it is empty, we don't need to delete the whole shebang
```